compiler/src/dmd/expression.d

```diff
@@ -3250,6 +3250,7 @@ extern (C++) final class AssocArrayLiteralExp : Expression
 
     Expressions* keys;
     Expressions* values;
+    Expression lowering; /// Lower to core.internal.newaa for static initializaton
```

Comment:
Why is the comment after the declaration

Reply:
DDoc knows the comment belongs to `lowering`

Reply:
It reads nicer preceding the declaration